elasticache/memcached/memcached.go

```diff
@@ -60,10 +63,16 @@ func NewMemcachedStack(scope constructs.Construct, id string, props *memcachedSt
 	// this prevents deletion errors caused by attempted subnet group deletes while the cluster still exists
 	memcachedCluster.AddDependency(subnetGroup)
 
+	arn := fmt.Sprintf("arn:aws:elasticache:%s:%s:cluster:%s", *stack.Region(), *stack.Account(), *elasticache.ResourceID(props.ClusterName, ""))
+	arn = strings.ToLower(arn)
```

Comment:
This seems a little silly but there doesn't seem to be a way to get it directly from the value returned by CDK. I checked and this is the correct value.